op-interop-mon/monitor/metric_collector.go

```diff
@@ -142,23 +199,22 @@ func (m *MetricCollector) CollectMetrics() {
 		}
 	}
 	// now we have the metrics consolidated, we can update the metrics
-	// executing messages
-	for chainID, blockNumberMap := range executingMessages {
-		for blockNumber, blockHashMap := range blockNumberMap {
-			for blockHash, statusMap := range blockHashMap {
-				for status, count := range statusMap {
-					m.log.Info("updating executing message stats", "chainID", chainID, "blockNumber", blockNumber, "blockHash", blockHash, "status", status, "count", count)
-					m.m.RecordExecutingMessageStats(chainID.String(), blockNumber, blockHash.String(), status, float64(count))
-				}
-			}
-		}
-	}
-	// initiating messages
-	for chainID, blockNumberMap := range initiatingMessages {
-		for blockNumber, statusMap := range blockNumberMap {
+	// message status
+	for executingChainID, initiatingChainMap := range messageStatus {
+		for initiatingChainID, statusMap := range initiatingChainMap {
 			for status, count := range statusMap {
-				m.log.Info("updating initiating message stats", "chainID", chainID, "blockNumber", blockNumber, "status", status, "count", count)
-				m.m.RecordInitiatingMessageStats(chainID.String(), blockNumber, status, float64(count))
+				m.log.Info("updating message status stats",
+					"executing_chain_id", executingChainID,
+					"initiating_chain_id", initiatingChainID,
+					"status", status,
+					"count", count,
+				)
```

Comment:
Might this get a little noisy and is it redundant with the metrics? Could bump it down to Debug?

Reply:
And/or extract the "status = invalid" case and turn that into a warning or error?

Reply:
Fair, this is already captured by metrics so we can just raise the invalid ones.

The total number of log lines would be `n * n * 3`, for `n` chains and 3 statuses. So 6 lines per second at the moment. Not a ton but could make log parsing harder I agree.